cloudinit/config/cc_yum_add_repo.py

```diff
@@ -58,72 +32,28 @@
 
 meta: MetaSchema = {
     "id": "cc_yum_add_repo",
-    "name": "Yum Add Repo",
-    "title": "Add yum repository configuration to the system",
-    "description": MODULE_DESCRIPTION,
-    "distros": distros,
-    "examples": [
-        dedent(
-            """\
-            yum_repos:
-              my_repo:
-                baseurl: http://blah.org/pub/epel/testing/5/$basearch/
-            yum_repo_dir: /store/custom/yum.repos.d
-            """
-        ),
-        dedent(
-            f"""\
-            # Enable cloud-init upstream's daily testing repo for EPEL 8 to
-            # install latest cloud-init from tip of `main` for testing.
-            yum_repos:
-              cloud-init-daily:
-                name: Copr repo for cloud-init-dev owned by @cloud-init
-                baseurl: {COPR_BASEURL}
-                type: rpm-md
-                skip_if_unavailable: true
-                gpgcheck: true
-                gpgkey: {COPR_GPG_URL}
-                enabled_metadata: 1
-            """
-        ),
-        dedent(
-            f"""\
-            # Add the file /etc/yum.repos.d/epel_testing.repo which can then
-            # subsequently be used by yum for later operations.
-            yum_repos:
-            # The name of the repository
-             epel-testing:
-               baseurl: {EPEL_TESTING_BASEURL}
-               enabled: false
-               failovermethod: priority
-               gpgcheck: true
-               gpgkey: file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL
-               name: Extra Packages for Enterprise Linux 5 - Testing
-            """
-        ),
-        dedent(
-            """\
-            # Any yum repo configuration can be passed directly into
-            # the repository file created. See: man yum.conf for supported
-            # config keys.
-            #
-            # Write /etc/yum.conf.d/my-package-stream.repo with gpgkey checks
-            # on the repo data of the repository enabled.
-            yum_repos:
-              my package stream:
-                baseurl: http://blah.org/pub/epel/testing/5/$basearch/
-                mirrorlist: http://some-url-to-list-of-baseurls
-                repo_gpgcheck: 1
-                enable_gpgcheck: true
-                gpgkey: https://url.to.ascii-armored-gpg-key
-            """
-        ),
+    "distros": [
```

Comment:
Thanks for the inlines of this parameter instead of declaring an unused local var outside this scope.